### PR TITLE
Community stats

### PR DESCRIPTION
**Pull request by stephend017**
## Overview
This is a big one. here is a list of what I did then followed by a list of what wasn't done.

## What was done
- added community ID to rating and meetup models
- added community stats model
- updated matchup algorithm to update community statistics when called
- updated accept and decline endpoints to update community statistics when called
- added endpoint for getting community statistics
- tested endpoint for getting community statistics

## What wasn't done
- tested community statistics were updated properly 
- update wiki (will do so after this is approved)
  - update rating wiki (model returned includes community now)
  - update meeting wiki (model returned includes community now)
  - create community stat wiki (very similar to join/leave)

I have not tested that statistics are updated properly because that requires I make some very complex unit tests for it since it occurs in another endpoint. I think it would be best to upgrade our testing infrastructure before doing so. That would make things easier. Please leave feedback and we can discuss further at todays meeting.

### CODE REVIEW

File: biit_server/meeting.py [hunk 1]
```diff
@@ -41,13 +42,15 @@ def __init__(
             duration = document_dict["duration"]
             location = document_dict["location"]
             meeting_type = document_dict["meettype"]
+            community = document_dict["community"]
```
Comment (lzhahn) [inline biit_server/meeting.py:45]: We may have to flush the DB as it can get mad if a key is not found.

File: biit_server/rating.py [hunk 2]
```diff
@@ -21,12 +24,18 @@ def __init__(self, meeting_id=None, document_snapshot=None, rating_dict=None):
             document_data = document_snapshot.to_dict()
             meeting_id = document_data.get("meeting_id")
             rating_dict = document_data.get("rating_dict")
+            community = document_data.get("community")
```
Comment (lzhahn) [inline biit_server/rating.py:27]: Again we may need to flush the db

**Review by lzhahn — COMMENTED** [review]
Looks good, communities in rating and meeting aren't part of the db rn so that may need to clear it in order for things to work.

File: biit_server/meeting_handler.py [hunk 3]
```diff
@@ -732,6 +808,23 @@ def matchup(request, auth):
             return http400("Rating id already taken")
 
     print(meeting_list)
```
Comment (fatcat2) [inline biit_server/meeting_handler.py]: print statement? can you do me a solid and remove this ty fam

**Review by fatcat2 — COMMENTED** [review]
We may need to flush the database. LGTM

**Review by fatcat2 — APPROVED** [review]
oops

**Review by fatcat2 — APPROVED** [review]
poggers in the chat brother let's mergerino lgtm

**Pull request merged**